terraform/grafana/Makefile

```diff
@@ -0,0 +1,19 @@
+tf_vars := TF_VAR_project=$(GCP_PROJECT_ID) 
+
+.PHONY: plan
+plan:
+	@ printf "Planning Terraform\n"
+	terraform fmt -recursive
+	terraform plan -out=.tfplan -compact-warnings -detailed-exitcode
```

Comment:
Can you add handling for different envs as is in the root (or the other [PR](https://github.com/e2b-dev/infra/pull/275/files#diff-991a410f5396dc41d1e4bec915569119944f9b0cc422a945d1ac249d1fc610e9))
```
ENV := $(shell cat .last_used_env || echo "not-set")
ENV_FILE := $(PWD)/.env.${ENV}
```